code/modules/marines/Marine_Weapons_V2.dm

```diff
@@ -189,6 +189,8 @@
 	ejectshell = 0 //Caseless
 	fire_delay = 4
 	slot_flags = SLOT_BACK
+	var
```

Comment:
Why are you changing the coding style? Why is this var defined over the span of two lines, and with an empty comment?